aptos-move/aptos-release-builder/data/enable_function_values.yaml

```diff
@@ -0,0 +1,14 @@
+remote_endpoint: ~
+name: "v1.33"
```

Comment:
I think the convention is to use some descriptive name, e.g., `v1.33-enable-function-values`